doc/manifest/schema/1.0.0/README.md

```diff
@@ -0,0 +1,12 @@
+# Windows Package Manager
+## Manifest Schema 1.0
+
+The Windows Package Manager 1.1 client does not support all fields in the 1.0 schema.
+
+Manifests submitted to the Windows Package Manager Community Repository should be submitted as a multi-file manifest. Only one version of a package may be submitted per pull request.
+
+A multi-file manifest contains:
+* One "version" file
```

Comment:
Can we get the links to the JSON schema here like you did for the 1.1.0 readme?

Reply:
incoming...